addons/web/static/src/views/list/list_controller.js

```diff
@@ -312,13 +311,9 @@ export class ListController extends Component {
         return list.isGrouped ? list.nbTotalRecords : list.count;
     }
 
-    get defaultExportList() {
-        return unique(
```

Comment:
FYI @aab-odoo We fixed recently the duplicated fields in https://github.com/odoo/odoo/pull/111808, but this is no longer needed to use `unique`, since fields are not read from the props.archInfo.columns, but from the rpc, which only returns "real" fields from the db